coral/src/domain/api-mocks/browser.ts

```diff
@@ -0,0 +1,5 @@
+import { setupWorker } from "msw";
+import { handlers } from "src/domain/api-mocks/handlers";
+
+// This configures a Service Worker with the given request handlers.
+export const worker = setupWorker(...handlers);
```

Comment:
As the handlers are supposed to be used only for missing APIs or other testing purposes, I would not setup those globally in this init. Rather, we could look into hooking up to an existing server initialized in `main.tsx` with [use()](https://mswjs.io/docs/api/setup-worker/use) and do the mocking where API call is made.

E.g. in the scope of this PR:

```
function mockUserAuthRequest() {
  window.msw.use(rest.post("/user/authenticate", (req, res, ctx) => {
    sessionStorage.setItem("is-authenticated", "true");

    const user: AuthUser = {
      name: "Super Admin",
    };
    return res(ctx.status(200), ctx.json(user));
  }));
}

const HomePage = () => {

  // REMOVE MOCK ONCE API EXISTS
  useEffect(mockUserAuthRequest, [])

  const { isLoading, data } = useQuery(["testData"], getAuthUser);
  return (
    <>
      <h1>Index</h1>
      {isLoading && <p>data is loading</p>}
      {!isLoading && data && <p>{data.name}</p>}
    </>
  );
};

export default HomePage;
```